File: 2.0/js/me/getApiCredential.js
'strict'

const CallfireClient = require('callfire-api-client-js');
const client = new CallfireClient('account-login', 'account-password');

client.ready(() => {
    client.me.getApiCredential({
      id: 379506003,
      // return only specific fields
      fields: 'name,enabled'
    })
      .then((response) => {
        console.log(response.obj);
      })
      .catch(function (err) {
        console.log('request error ' + err.data);
      });
  },
  (clientError) => {
    console.log('client error ' + clientError);
  }
);

File: 2.0/python/me/getApiCredential.py
from callfire.client import CallfireClient

client = CallfireClient('account-login', 'account-password')
response = client.me.getApiCredential(
    id=379506003,
    # return only specific fields
    fields='name,enabled'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)
